### DISCUSSION TITLE
Can I express _TABLE_SUFFIX?

### DISCUSSION THREAD

**Discussion by tkawachi** [EvgSkv/logica#41] [Q&A] [answered]
I'm interested in the possibility of writing SQL in a way that is easy to understand. I was thinking of rewriting an existing query, but how should I express `_TABLE_SUFFIX`?

**Comment by EvgSkv** [comment, marked as answer]
Hello,
thanks for your interest!
`_table_suffix` is exposed as a column, thus it can be used as a field of the predicate as such:

```python
Q(s) += 1 :- `bigquery-public-data.noaa_gsod.gsod*`(_table_suffix: s)
```

Does this answer your question?

**Reply by tkawachi** [reply]
Yes. Thank you very much. I will try to touch it more to understand how to use it.
